src/epiphany.gresource.xml

```diff
@@ -25,4 +28,8 @@
     <file compressed="true">prism.css</file>
     <file compressed="true">prism.js</file>
   </gresource>
+  <gresource prefix="/org/gnome/Epiphany/icons">
+    <file compressed="true" alias="scalable/actions/ephy-bookmark-tag-symbolic.svg">icons/hicolor/scalable/actions/ephy-bookmark-tag-symbolic.svg</file>
```

Comment:
Ah, so you're already using an alias. You shouldn't need the deep directory structure in the git repo.